spec/features/user_can_delete_reviews_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+feature 'user can delete their own reviews', %{
+  As a user,
+  I want to delete my own reviews,
+  So that no one can see them
+} do
+
+  # Acceptance Criteria
+  # [√] I can delete my reviews from my profile page
+  # [√] If I delete a review, no one can see them anymore
+
+  let!(:user) { FactoryGirl.create(:user) }
+  let!(:author) { FactoryGirl.create(:author) }
+  let!(:genre) { FactoryGirl.create(:genre) }
+  let!(:book) { FactoryGirl.create(
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.